.travis.yml

```diff
@@ -9,21 +8,21 @@ matrix:
       env: CHECK_DOCS=1
     - python: 3.8
       env: CHECK_FORMATTING=1
-    # The pypy tests are slow, so list them early
-    - python: pypy3
+    # The pypy tests are slow, so we list them first
+    - python: pypy3.6-7.2.0
+      dist: bionic
+    - language: generic
+      env: PYPY_NIGHTLY_BRANCH=py3.6
     # Uncomment if you want to test on pypy nightly:
     # - language: generic
     #   env: USE_PYPY_NIGHTLY=1
-    - python: 3.6
+    - python: 3.6-dev
     # As of 2018-07-05, Travis's 3.7 and 3.8 builds only work if you
     # use dist: xenial AND sudo: required
     # See: https://github.com/python-trio/trio/pull/556#issuecomment-402879391
-    - python: 3.7
-      dist: xenial
-      sudo: required
-    - python: 3.8
-      dist: xenial
-      sudo: required
+    - python: 3.7-dev
+    - python: 3.8-dev
+    - python: 3.9-dev
```

Comment:
For what it's worth, the reason Trio's config is like that is because most tests happen in GitHub Actions. But since Travis has good support for dev versions and PyPy, Trio continues to test those in Travis.

In trimeter's case, it would be better to keep both 3.x and 3.x-dev for each supported version

Reply:
Thanks for the explanation.  I might try reworking QTrio's GHA configuration first.  There's a fair bit of repetition in the way it is now that isn't obviously useful in my head vs. the one big matrix I made [over in pytest-twisted](https://github.com/pytest-dev/pytest-twisted/blob/37693ec18f41216e8255e761b9dbfdd149fbc705/.github/workflows/ci.yml) for example.  But maybe as I implement that in QTrio I'll find an issue, of course.